builds.yml

```diff
@@ -145,7 +145,7 @@ env:
   # The unlock password
   - PASSWORD: null
   - TEST_SRP: null
-  - SKIP_ONBOARDING: null
```

Comment:
we changed this to a more generic name, as the feature is expanded to handle more wallet state data, than just skipping the onboarding